ffmpeg/ffmpeg.go

```diff
@@ -255,20 +269,21 @@ func GetCodecInfo(fname string) (CodecStatus, string, string, PixelFormat, error
 	params_c.pixel_format = C.AV_PIX_FMT_NONE
 	status := CodecStatus(C.lpms_get_codec_info(cfname, &params_c))
 	if C.strlen(acodec_c) < 255 {
-		acodec = C.GoString(acodec_c)
+		format.Acodec = C.GoString(acodec_c)
 	}
 	if C.strlen(vcodec_c) < 255 {
-		vcodec = C.GoString(vcodec_c)
+		format.Vcodec = C.GoString(vcodec_c)
 	}
-	pixelFormat := PixelFormat{int(params_c.pixel_format)}
-	return status, acodec, vcodec, pixelFormat, nil
+	format.PixFormat = PixelFormat{int(params_c.pixel_format)}
+	format.Width = int(params_c.width)
+	format.Height = int(params_c.height)
+	return status, format, nil
 }
 
 // GetCodecInfo opens the segment and attempts to get video and audio codec names. Additionally, first return value
 // indicates whether the segment has zero video frames
-func GetCodecInfoBytes(data []byte) (CodecStatus, string, string, PixelFormat, error) {
-	var acodec, vcodec string
-	var pixelFormat PixelFormat
+func GetCodecInfoBytes(data []byte) (CodecStatus, MediaFormatInfo, error) {
```

Comment:
Hi @AlexKordic ,  This change doesn't work go-livepeer, see [here](https://github.com/livepeer/go-livepeer/blob/613e60b5d88bdee6ecfd49650488375e29f2455a/server/mediaserver.go#L820)

Reply:
Hi @oscar-davids, merging this one https://github.com/livepeer/go-livepeer/pull/2466 shortly